components/scream/src/physics/shoc/shoc_diag_second_moments_srf_impl.hpp

```diff
@@ -0,0 +1,44 @@
+
+#ifndef SHOC_DIAG_SECOND_MOMENTS_SRF_IMPL_HPP
+#define SHOC_DIAG_SECOND_MOMENTS_SRF_IMPL_HPP
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+#include "physics_functions.hpp" // also for ETI not on GPUs
+
+namespace scream {
+namespace shoc {
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::shoc_diag_second_moments_srf(
+    const Int& shcol, 
+    const Spack& wthl_sfc, const Spack& uw_sfc, const Spack& vw_sfc,
+    Spack& ustar2, Spack& wstar)
+{
+ // Purpose of this subroutine is to diagnose surface
```

Comment:
We use 2-space indents. This looks like only 1-space.

Reply:
Can we fix this now instead of later? (In case it's copy-pasted elsewhere).